src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/FrameworkEventSource.cs

```diff
@@ -114,6 +116,12 @@ public unsafe void ThreadPoolDequeueWorkObject(object workID)
             ThreadPoolDequeueWork((long)*((void**)Unsafe.AsPointer(ref workID)));
         }
 
+        [Event(32, Level = EventLevel.Verbose, Keywords = Keywords.ThreadPool)]
```

Comment:
Not sure how the event IDs are assigned. I looked up what CoreFX and NetFX had and used the next available in the row.

Reply:
The only important thing about the ID is that it is unique.  The only thing that makes it at all tricky is that we have two version of this file (.NET Core and .NET Desktop), and we want it to be unique considering both of them.  As long as we add events to .NET Core every time we do to Desktop, then everything is OK (and I took a look and this seems to be the case.   

The long and the short of this is that 32 is OK to use.   

Could you put a comment that we should be using the IDs in the range from 33 to 149 first for any new events.